src/evict/evict_lru.c

```diff
@@ -2523,29 +2537,30 @@ __verbose_dump_cache_single(
           __wt_msg(session,
             "internal: "
             "%" PRIu64 " pages, "
-            "%" PRIu64 "MB, "
+            "%" PRIu64 "KB, "
             "%" PRIu64 "/%" PRIu64 " clean/dirty pages, "
-            "%" PRIu64 "/%" PRIu64 " clean/dirty MB, "
-            "%" PRIu64 "MB max page, "
-            "%" PRIu64 "MB max dirty page",
-            intl_pages, intl_bytes / WT_MEGABYTE, intl_pages - intl_dirty_pages, intl_dirty_pages,
-            (intl_bytes - intl_dirty_bytes) / WT_MEGABYTE, intl_dirty_bytes / WT_MEGABYTE,
-            intl_bytes_max / WT_MEGABYTE, intl_dirty_bytes_max / WT_MEGABYTE));
+            "%" PRIu64 "/%" PRIu64 " clean/dirty KB, "
+            "%" PRIu64 "KB max page, "
+            "%" PRIu64 "KB max dirty page",
+            intl_pages, intl_bytes / WT_KILOBYTE, intl_pages - intl_dirty_pages, intl_dirty_pages,
+            (intl_bytes - intl_dirty_bytes) / WT_KILOBYTE, intl_dirty_bytes / WT_KILOBYTE,
+            intl_bytes_max / WT_KILOBYTE, intl_dirty_bytes_max / WT_KILOBYTE));
```

Comment:
Note: I will revert this, it's just unfortunate that some of the existing tests can hang with sizes that round to 0MB.

Reply:
I don't feel strongly - we could keep the change.